kimchi/src/verifier.rs

```diff
@@ -278,7 +278,7 @@ where
         //~~ - poseidon selector
         //~~ - the 15 register/witness
         //~~ - 6 sigmas evaluations (the last one is not evaluated)
-        for (p, e) in p_eval.iter().zip(&self.evals) {
+        for (p, e) in p_eval.iter().zip(&self.evals.array()) {
```

Comment:
same here, we don't need to call `array`

Reply:
Yeah... This solution with the array is the one with fewer modifications in the original code. But if we are fine with being more explicit about zeta and zeta_omega, then I can change it.